### ISSUE TITLE
It would be better if it supports more base types

### ISSUE THREAD

**Issue by xcvista** [ColinEberhardt/LinqToObjectiveC#11]
Including `NSSet`, `NSOrderedSet`, `NSPointerArray`, `NSHashMap` as well as their CoreFoundation counterparts.

I am creating a new LINQKit that took some code from your project but also provides such diversity in type support, as well as CoreFoundation with and without blocks.

**Comment by ColinEberhardt** [comment]
Yeah, I agree - more base types would be good. Although I don't have time to add them myself.

Good luck with LinqKit!
